sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2143,6 +2144,8 @@ public static <T> Write<T> write() {
         .setMaxRetryJobs(1000)
         .setPropagateSuccessfulStorageApiWrites(false)
         .setDirectWriteProtos(true)
+        .setDefaultMissingValueInterpretation(
+            AppendRowsRequest.MissingValueInterpretation.DEFAULT_VALUE)
```

Comment:
Is DEFAULT_VALUE our current default, or do we default to null?

Reply:
I just want to double check we aren't breaking backwards compatability

Reply:
Yeah -  the BigQuery sink didn't really support default values before, so I consider the previous behavior as accidental, undefined behavior. Doesn't mean that customers didn't rely on it, but I'm ok with changing it especially since we're still providing a way to default to null.